### PR TITLE
settings acceptance tests

### PR DESCRIPTION
**Pull request by mahmoudyousef98**
***For All Submissions:***

*Proposed Changes:*

Changed the Iteration 1 README to include acceptance tests for set_settings() and get_settings()

*Testing Instructions:*

* [ ] Does simply running `pytest` suffice?

**Finally**,
* [ ] Have you updated the version number, by running `git tag v0.x`?

------

**For new features:**
* [ ] If you have used new packages, have you remade `requirements.txt` (explained in README)?

**For new tests:**
* [ ] Have you made sure your tests are in the `tests/` dir?
* [ ] Have you made sure your tests are in a file that starts with '`test_`'?
* [ ] Have you made sure your test functions starts with '`test_`'?

### CODE REVIEW

File: docs/README_iteration1.md [hunk 1]
```diff
@@ -44,6 +44,27 @@ By running `python src/start.py`, you can start the GUI for yourself and try to
...
+
+- test_set_settings()
+    - set_settings(self, values, path) takes in two parameters: a dictionary of settings and a file path to the video. It will validate the settings and the validity of the path and return True if the settings are valid (as well as actually set the settings so the Job class can work) or False otherwise.
+    - "values" is a dictionary path containing the following elements:
+        - "conf": the confidence interval of acceptance of a frame classification. It must be a value between 0 and 1
+        - "poll": the rate of how we pull frames to classify. It must be an integer value greater than 0
+        - "anti": the amount of time in seconds where we do not accept a frame classification (giving us the bounds of the clip). It must be an integer value greater than 0
+        - "runtime": the runtime of the video in seconds. Must be an integer greater than 0
+        - "search": an array of search terms to use. It must be an array of strings
+    - "path" is an OS path to the video to import. Pyhton must be able to open the path.
```
Comment (suchak1) [inline docs/README_iteration1.md]: Python*

Comment (suchak1) [reply docs/README_iteration1.md]: also nice job

**Comment by suchak1** [comment]
nice job

File: docs/README_iteration1.md (hunk 1 above)
Comment (mahmoudyousef98) [reply docs/README_iteration1.md]: My bad, I'll fix that right now

**Pull request merged**